react/CurrentSellerContext.tsx

```diff
@@ -30,3 +30,5 @@ export const CurrentSellerProvider: FC<ProviderProps> = ({
     </CurrentSellerContext.Provider>
   )
 }
+
+export default CurrentSellerContext
```

Comment:
Why is that necessary? Why not just import the CurrentSellerPovider?

You can import it as:

```
import { CurrentSellerProvider } from 'vtex.seller-selector/CurrentSellerContext'
```

Reply:
@klzns Sorry I didn't try this method to import the contexts, its worked fine here! Thank you!

Reply:
No problem! ✌️

Reply:
@klzns Actually... I used this way to import the Contexts but isn't working, just `vtex link` pass with success, but the component on front gives me this error:

![image](https://user-images.githubusercontent.com/49173685/86969476-4226d700-c144-11ea-8680-b013f10e6c40.png)

Code example:
![image](https://user-images.githubusercontent.com/49173685/86969518-510d8980-c144-11ea-8522-575fc5d950e6.png)

Reply:
@klzns some news about that?

Reply:
Sorry, I'll get to you in a second